CONTRIBUTING.md

```diff
@@ -1,44 +1,26 @@
-![Toolbox logo](data/logo/toolbox-logo-landscape.svg)
+![Toolbox logo](data/gfx/powerup.gif)
 
 # Contributing to Toolbox
 
 Thank you for wanting to contribute to Toolbox! We greatly appreciate your
 interest!
 
-# Table of contents
-
-- [Contributing to Toolbox](#contributing-to-toolbox)
-- [Table of contents](#table-of-contents)
-- [Reporting Bugs](#reporting-bugs)
-  - [Before Submiting a Bug Report](#before-submiting-a-bug-report)
-  - [Writing a Bug Report](#writing-a-bug-report)
-- [Making Suggestions](#making-suggestions)
-  - [Before Submitting a Suggestion](#before-submitting-a-suggestion)
-  - [Writing a Suggestion](#writing-a-suggestion)
-- [First Contribution](#first-contribution)
-- [Pull Requests](#pull-requests)
-  - [Creating a Pull Request](#creating-a-pull-request)
-  - [After Creating a Pull Request](#after-creating-a-pull-request)
-- [Little Style Guide](#little-style-guide)
-
 # Reporting Bugs
 
-## Before Submiting a Bug Report
+## Before Submitting a Bug Report
 
 - Check if your issue is already reported in our [bug tracker](https://github.com/containers/toolbox/issues)
   - If the issue is already reported and is marked as **OPEN**, comment on it
     and if possible and needed, share info about the issue just as if you were
-    submiting a new issue
+    submitting a new issue
   - If the issue is marked as **CLOSED**, check if your version of Toolbox is
     up-to-date or if there are some steps, described in the closed issue, that
     you should follow. If you are still experiencing the issue, please file a
     new issue
 - See our [documentation](https://docs.fedoraproject.org/en-US/fedora-silverblue/toolbox/)
   if there are some steps that could help you solve your issue
```

Comment:
We trying to consolidate our documentation around our [website](https://containertoolbx.org/), our manuals, and this code repository.  Linking to another documentation source goes against that.  I'd suggest linking to the [top-level manual](https://github.com/containers/toolbox/blob/main/doc/toolbox.1.md)

Reply:
Good point. Adding to my backlog.